llvm/test/Transforms/Inline/inline-recursive-fn2.ll

```diff
@@ -0,0 +1,45 @@
+; RUN: opt -passes='cgscc(inline),instcombine,cgscc(inline)' -S -debug-only=inline -disable-output < %s 2>&1 | FileCheck %s
```

Comment:
Needs `REQUIRES: asserts` if you're matching debug output. Though I'm not super clear on what exactly this particular test is intended to check.